app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -813,6 +791,63 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
         }
     }
 
+    override fun onHomePressed(): Boolean {
+        if (pipFeature.onHomePressed()) {
+            return true
+        }
+        return false
+    }
+
+    private fun pipModeChanged(enabled: Boolean) {
+        val sessionManager = requireComponents.core.sessionManager
+        val customTabSession = customTabSessionId?.let { sessionManager.findSessionById(it) }
+        val session = customTabSession ?: sessionManager.selectedSession
```

Comment:
We could use our working SessionManager ktx here as well! 😄

Reply:
👍